template/ROV.html

```diff
@@ -62,7 +62,7 @@
             <div id="c0" class="screen" onclick="switching('camera_0')">
                 <div class="title">Camera 0</div>
                 <!--<img id="camera_0" class="rotate180 rotate90" src="file:///home/vboxuser/Desktop/EVA_2025/gui/template/ROV_files/stream_002">-->
-                <img id="camera_0" class = "video-container" src="http://10.0.0.254:8080/stream" muted autoplay loop></video>
+                <img id="camera_0" class = "video-container" src="" muted autoplay loop></video>
```

Comment:
```suggestion
                <img id="camera_0" class = "video-container" src=""  />
```

"muted autoplay and loop" are parameters for video tags, is this tag an img or a video? I think we can use `<img />` syntax above

Reply:
sounds great.